tests/benchmark/test_worst_stateful_opcodes.py

```diff
@@ -529,8 +614,15 @@ def test_worst_selfdestruct_existing(
     final_storage_gas = (
         gas_costs.G_STORAGE_RESET + gas_costs.G_COLD_SLOAD + (gas_costs.G_VERY_LOW * 2)
     )
-    base_costs = intrinsic_gas_cost_calc() + (gas_costs.G_VERY_LOW * 4) + final_storage_gas
+    memory_expansion_cost = fork().memory_expansion_gas_calculator()(new_bytes=96)
+    base_costs = (
+        intrinsic_gas_cost_calc()
+        + (gas_costs.G_VERY_LOW * 12)  # 8 PUSHs + 4 MSTOREs
```

Comment:
Nice, did you find these missing MSTOREs in the calc thanks to the new assertion feature?

Reply:
Yes exactly! I spent quite some time yesterday trying to debug these failing tests due to the new constrain, but in the end I feel is good to have this information well explained in the test logic because if we decide to change something in the future, having the exact source of all gas costs is a nice starting point to have.

Reply:
Agree!